src/index.js

```diff
@@ -26,13 +26,15 @@ type Props = {
   float?: ?ReactElement<any>;
   options?: ?Options;
   zIndex?: ?number|string;
+  className?: string; 
```

Comment:
`className?:` means the prop is optional; `?string` would allow the prop to be present but null/undefined. Let's make it `?string` for consistency.

Reply:
Let's make it specific what the class name applies to: maybe we should name the new prop "floatContainerClassName". It's wordy, but accurate. Don't want to use the word "portal" because that word isn't used in the documentation or props names. Don't want it just to be "float" because then it implies the class name may be applied to the element passed as the float prop. The concept of the "float element's container" is already used in the documentation. The float element's container should just be an implementation detail for most uses of react-float-anchor; the verbosity of the name maybe usefully implies to the user that they're straying off of the paved trail.

Reply:
I agree with making it more explicit (just thought of the `FloatAnchor` as the actual portal when implementing it... it was late). However why would we allow having a `floatContainerClassName ` set and it not being a `string` (if I understand you correctly)? We'd have to check before setting it on the DOMNode (which I thought flow and proptypes is for). IMHO the prop being present but `undefined`/`null` might in 90% be the result of an error in the component that renders the anchor (sth that evaluates to `undefined` and should probably have been a className string). I'd like to catch that

Reply:
If the caller doesn't pass in floatContainerClassName, then `this.props.floatContainerClassName` is undefined, exactly as if they had the prop with undefined as its value. The javascript as you have it now (with `|| ''`) handles both the cases of the prop being passed as nullish or not being passed at all, but the Flow type as it is now makes a distinction.

I'm mainly a stinker about this because allowing nullish values means it's easier to write a component that wraps FloatAnchor and passes a value from its props (whether or not it's present) through to FloatAnchor as `floatContainerClassName` prop.